web/src/components/Plugin/PluginDetail.tsx

```diff
@@ -247,7 +249,7 @@ const PluginDetail: React.FC<Props> = ({
   return (
     <>
       <Drawer
-        title={`Plugin: ${name}`}
+        title={formatMessage({ id: 'component.plugin.editor' })}
```

Comment:
It would be more clear if we can show the plugin name in the title, which will  tell user what plugin they are configuring